src/node/utxo_snapshot.h

```diff
@@ -28,16 +28,17 @@ class Chainstate;
 namespace node {
 //! Metadata describing a serialized version of a UTXO set from which an
 //! assumeutxo Chainstate can be constructed.
+//! All metadata fields come from an untrusted file, so must be validated
```

Comment:
nit: could indicate in the commit message `ShapShotMetadata` version was also bumped.

Reply:
done